### DISCUSSION TITLE
图表示例的菜单栏不好用,有什么可以改进的吗?

### DISCUSSION THREAD

**Discussion by Plortinus** [antvis/G2Plot#2614] [General]
![Dingtalk_20210608101223](https://user-images.githubusercontent.com/20693993/121111800-1aebf480-c842-11eb-87d2-9d040a70ad79.jpg)

个人感觉,这个菜单栏用起来不太好用
菜单项多,切换其他示例的时候需要滚轮好久.
不如https://g2plot.antv.vision/zh/examples/gallery 这个的侧边栏好用.
不知道有什么改进的办法

**Comment by visiky** [comment]
之前的开发同学不再跟进了...
我们会在 8 月份左右安排新的同学来优化这个问题：https://github.com/antvis/gatsby-theme-antv/issues/291

**Reply by visiky** [reply]
已经支持
